devel-common/src/tests_common/test_utils/db.py

```diff
@@ -351,6 +353,7 @@ def clear_all():
     clear_db_pools()
     clear_db_connections(add_default_connections_back=True)
     clear_db_deadline()
-    clear_dag_specific_permissions()
+    if "FabAuthManager" in conf.get("core", "auth_manager"):
+        clear_dag_specific_permissions()
```

Comment:
Doing it in the function would avoid checking each time we are calling this function